test/validation/api/time/time.c

```diff
@@ -423,11 +423,12 @@ static void time_test_accuracy(time_cb time_cur, time_from_ns_cb time_from_ns)
 {
 	int i;
 	odp_time_t t1, t2, wait, diff;
-	clock_t c1, c2;
+	struct timeval tv1, tv2, tvdiff;
 	double sec_t, sec_c;
 	odp_time_t sec = time_from_ns(ODP_TIME_SEC_IN_NS);
 
-	c1 = clock();
+	i = gettimeofday(&tv1, NULL);
```

Comment:
gettimeofday() may be changed on NTP/PTP and it's not good for measuring intervals.  clock_gettime(CLOCK_MONOTONIC, ...) I think will be good one here.

Reply:
also please add link to bugz if it's bug fix to git comment.

Reply:
Nice idea, I will update this PR.

Reply:
@muvarov maybe we can reenable time_main test in Travis with this fix? What do you think wrt [previous issue](https://bugs.linaro.org/show_bug.cgi?id=3017)